gateway/1.4/modules/ROOT/pages/flex-view-replica-status.adoc

```diff
@@ -5,21 +5,45 @@ endif::[]
 :page-aliases: runtime-manager::flex-gateway-replica-status.adoc
 :imagesdir: ../assets/images
 
-To view information about the status of Flex Replicas:
+In Runtime Manager, you can view the status of groups of Flex Gateway replicas and individual replicas.
+
+== Groups of Flex Gateway Replicas
+
+To view information about the status of groups of Flex Gateway replicas:
 
 . From Anypoint Platform, select *Runtime Manager*.
-. Select *Flex Gateway* in the navigation menu.
+. Select *Flex Gateway* in the navigation menu. +
+By default, all Flex Gateways are shown. To filter by status, click the *Any Status* arrow and the status you want to filter.
+
+The *Name* column in the list of Flex Gateways lists the name and the number of connected replicas.
```

Comment:
```suggestion
The *Name* column lists the name and the number of connected replicas.
```
It seems implied that they're already looking at the list of gateways. Also, "list" is used twice in the sentence, which seems wordy.